normalize.css

```diff
@@ -0,0 +1,427 @@
+/*! normalize.css v6.0.0 | MIT License | github.com/necolas/normalize.css */
```

Comment:
That's not the latest version of [Normalize.css](http://necolas.github.io/normalize.css/). Probably, you'd want to use v7.